### PR TITLE
Add Android build documentation and Application customisation details

### PR DESCRIPTION
**Pull request by krishnakumar4a4**
This PR contains following documentation content
1. Android Dev build
2. Customisation
  - Workflow customisation
  - UI customidation
  - Locale customisation

### CODE REVIEW

File: docs/customisation/workflow-customisation.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+This state machine takes care of actions related to storing and retrieving data on the mobile phone. It exposes wrapper to all the other state machines to work with data stored on the device. It also performs the custom encryption and decryption required for saving and retrieving data from the underlying store. As of now, the store state machine uses a library called [async-storage](https://react-native-async-storage.github.io/async-storage/docs/api) which abstracts how data is stored between IOS and Android. 
+
+## auth.ts
+This state machine helps to setup authentication for the application. It allows user to choose between biometric and passcode based authetication on the first launch.
+
+## vc.ts
+This state machine helps to show a couple of screens related to VC. One is My VCs where all user downloaded VCs will be shown. Another one is Received VCs where all received VCs can be shown.
+
+## vcItem.ts
+TODO:
```
Comment (krishnakumar4a4) [inline docs/customisation/workflow-customisation.md:27]: @MonobikashDas Please add content for this component

File: docs/customisation/workflow-customisation.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## request.ts
+This state machine is instantiated when the user launches the verification section which display QR code. This QR is generated with content from low level [offline VC sharing component](../architecture/components.md#offline-vc-sharing-component). The content of the QR code is scanned by a wallet inji application to establish connection with verifier and share the VC credential. Once the VC is received on the Verifier side, the state machine allows to see the received VC details as well for Verification.
+
+## scan.ts
+This state machine is instantiated when the user launches the scanner section which opens up camera to scan the QR code presented by the Verifier. The scanned data is fed into underlying [offline VC sharing component](../architecture/components.md#offline-vc-sharing-component) to allow discovery of the Verifier device and establishing connnection to it. Once the connection is established, the user is allowed select the downloaded VCs that can be shared to Verifier. The state machine also allows selfie/face verification before sharing VC.
+
+## revoke.ts
+TODO:
```
Comment (krishnakumar4a4) [inline docs/customisation/workflow-customisation.md:42]: @MonobikashDas Please add content for this component

**Comment by krishnakumar4a4** [comment]
@MonobikashDas As discussed, we have added the first draft of the content. 
I have readily tagged you on couple of places where you can add content. If you feel we need to add more details please add it accordingly.

@ravikp please add your inputs on Build related documentation.

**Review by SwatiPeriwal — APPROVED** [review]
Merging changes

**Pull request merged**